build.gradle

```diff
@@ -31,14 +31,13 @@ msbuild {
 
 nunit {
     exclude 'LongRunning'
-    testAssemblies = [ msbuild.projects['ILRepack.Tests'].properties.TargetPath ] + (OperatingSystem.current().unix ?
-                     [] : [ msbuild.projects['ILRepack.IntegrationTests'].properties.TargetPath ])
+    testAssemblies = [ msbuild.projects['ILRepack.Tests'].properties.TargetPath + msbuild.projects['ILRepack.IntegrationTests'].properties.TargetPath ]
```

Comment:
looks like we shouldn't concat but just "add" via "," ? That seems to be the exception for the appveyor build